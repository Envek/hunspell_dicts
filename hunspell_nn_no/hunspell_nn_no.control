# hunspell_nn_no extension
comment = 'Norwegian (norsk) Hunspell Dictionary'
default_version = '1.0'
module_pathname = '$libdir/hunspell_nn_no'
relocatable = true