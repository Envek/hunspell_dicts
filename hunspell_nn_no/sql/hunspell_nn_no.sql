CREATE EXTENSION hunspell_nn_no;

SELECT ts_lexize('norwegian_hunspell', 'kjem');
SELECT ts_lexize('norwegian_hunspell', 'heiter');
SELECT ts_lexize('norwegian_hunspell', 'guten');
SELECT ts_lexize('norwegian_hunspell', 'gutar');
SELECT ts_lexize('norwegian_hunspell', 'kyrkjene');
SELECT ts_lexize('norwegian_hunspell', 'fotballklubber');
SELECT ts_lexize('norwegian_hunspell', 'klubbfotballen');
SELECT ts_lexize('norwegian_hunspell', 'stavekontroll');
