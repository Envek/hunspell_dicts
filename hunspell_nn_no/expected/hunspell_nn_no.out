CREATE EXTENSION hunspell_nn_no;
SELECT ts_lexize('norwegian_hunspell', 'kjem');
   ts_lexize   
---------------
 {kjem,kjemme}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'heiter');
  ts_lexize   
--------------
 {heite,heit}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'guten');
  ts_lexize  
-------------
 {guten,gut}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'gutar');
 ts_lexize 
-----------
 {gut}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'kyrkjene');
 ts_lexize 
-----------
 {kyrkje}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'fotballklubber');
           ts_lexize            
--------------------------------
 {fotball,klubb,fot,ball,klubb}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'klubbfotballen');
                          ts_lexize                          
-------------------------------------------------------------
 {klubbfotball,klubb,fotball,klubb,fot,balle,klubb,fot,ball}
(1 row)

SELECT ts_lexize('norwegian_hunspell', 'stavekontroll');
             ts_lexize             
-----------------------------------
 {stave,kontroll,stave,kont,rolle}
(1 row)

